src/Nfts/Nfts.tsx

```diff
@@ -4,23 +4,31 @@ import {RefreshControl, ScrollView, StyleSheet, Text, View} from 'react-native'
 import {SafeAreaView} from 'react-native-safe-area-context'
 
 import {Icon, Spacer} from '../components'
-import {useFilteredNfts} from './hooks'
+import {useSearch} from '../Search/SearchContext'
+import {useSelectedWallet} from '../SelectedWallet'
+import {useNfts} from '../yoroi-wallets'
+import {filterNfts} from './filterNfts'
 import {ImageGallery, SkeletonGallery} from './ImageGallery'
 import {useNavigateTo} from './navigation'
 import {NoNftsScreen} from './NoNftsScreen'
 
 export const Nfts = () => {
   const navigateTo = useNavigateTo()
-  const handleNftSelect = (index: number) => navigateTo.nftDetails(nfts[index].id)
   const [isManualRefreshing, setIsManualRefreshing] = React.useState(false)
+  const wallet = useSelectedWallet()
   const strings = useStrings()
 
-  const {search, nfts, isLoading, refetch, isError} = useFilteredNfts({
+  const {isLoading, nfts, refetch, isError} = useNfts(wallet, {
     onSettled: () => {
       if (isManualRefreshing) setIsManualRefreshing(false)
     },
   })
 
+  const {search: nftsSearchTerm} = useSearch()
+  const nftsSearchResult = filterNfts(nftsSearchTerm, nfts).sort((NftA, NftB) => NftA.name.localeCompare(NftB.name))
```

Comment:
save capitals for classes/modules, still I'll be fine with a,b.
```suggestion
  const nftsSearchResult = filterNfts(nftsSearchTerm, nfts).sort((nftA, nftB) => nftA.name.localeCompare(nftB.name))
```

Reply:
sorting after searching, implies that only filtered nfts will be sorted, I'd suggest splitting into `sortedNfts` and applying the filter on top of the sorted nfts, or even sorted right before adding the nfts into the data for the list.

Reply:
why sort nfts that are not displayed?

Reply:
yap, that makes sense, sort for last right before adding to the list component.